monkey/infection_monkey/telemetry/tests/test_technique_telems.py

```diff
@@ -0,0 +1,185 @@
+import pytest
+
+from common.utils.attack_utils import ScanStatus, UsageEnum
+from infection_monkey.model import VictimHost
+from infection_monkey.telemetry.attack.t1005_telem import T1005Telem
+from infection_monkey.telemetry.attack.t1035_telem import T1035Telem
+from infection_monkey.telemetry.attack.t1064_telem import T1064Telem
+from infection_monkey.telemetry.attack.t1105_telem import T1105Telem
+from infection_monkey.telemetry.attack.t1106_telem import T1106Telem
+from infection_monkey.telemetry.attack.t1107_telem import T1107Telem
+from infection_monkey.telemetry.attack.t1129_telem import T1129Telem
+from infection_monkey.telemetry.attack.t1197_telem import T1197Telem
+from infection_monkey.telemetry.attack.t1222_telem import T1222Telem
+
+
+GATHERED_DATA_TYPE = '[Type of data collected]'
+INFO = '[Additional info]'
+MACHINE = VictimHost('127.0.0.1')
+STATUS = ScanStatus.USED
+USAGE = UsageEnum.SMB
+SRC_IP = '0.0.0.0'
+DST_IP = '0.0.0.1'
+FILENAME = 'virus.exe'
+PATH = 'path/to/file.txt'
+COMMAND = 'echo hi'
+
+
+@pytest.fixture
+def T1005_telem_test_instance():
+    return T1005Telem(STATUS, GATHERED_DATA_TYPE, INFO)
+
+
+def test_T1005_telem_category(T1005_telem_test_instance):
+    assert T1005_telem_test_instance.telem_category == 'attack'
+
+
+def test_T1005_get_data(T1005_telem_test_instance):
+    actual_data = T1005_telem_test_instance.get_data()
+    expected_data = {'status': STATUS.value,
+                     'technique': 'T1005',
+                     'gathered_data_type': GATHERED_DATA_TYPE,
+                     'info': INFO}
+    assert actual_data == expected_data
+
+
+@pytest.fixture
+def T1035_telem_test_instance():
+    return T1035Telem(STATUS, USAGE)
+
+
+def test_T1035_telem_category(T1035_telem_test_instance):
+    assert T1035_telem_test_instance.telem_category == 'attack'
+
+
+def test_T1035_get_data(T1035_telem_test_instance):
+    actual_data = T1035_telem_test_instance.get_data()
+    expected_data = {'status': STATUS.value,
+                     'technique': 'T1035',
+                     'usage': USAGE.name}
+    assert actual_data == expected_data
+
+
+@pytest.fixture
+def T1064_telem_test_instance():
```

Comment:
Do we know why this telemetry is a subclass of `AttackTelem` and not `UsageTelem`?